.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.2.3
+current_version = 0.2.4
```

Comment:
wouldn't it be a more major version update, since changing from minimum 3.6 to 3.7 and I see deprecated functions getting removed.